extension/src/main/scala/io/scalac/mesmer/extension/upstream/OpenTelemetryHttpMetricsMonitor.scala

```diff
@@ -58,31 +58,32 @@ final class OpenTelemetryHttpMetricsMonitor(
   import HttpMetricsMonitor._
 
   private lazy val requestTimeRequest = meter
-    .longValueRecorderBuilder(metricNames.requestDuration)
+    .histogramBuilder(metricNames.requestDuration)
```

Comment:
TODO: This causes our grafana example dashboards to fail when calculating max values. Figure out what can we do about this

Reply:
Ok. Now I know what is going on here. We were using the long value recorder as it was the only meter type that allowed us to register all (?) samples as they were, without any aggregation. Thanks to that we were able to calculate: 
 - avg (just take the sum of all response times and divide it by the whole count)
 - max (just take the biggest response time as it is there in the value recorder)
 
In 1.7.0 the `Meter` interface is different and no longer allows creating Value recorders. What we're left with are histograms.  Not that it's not sensible to use them for collecting the response times, it's just that they work a little bit differently. What they track is: 
 - the sum of all observation values (per bucket/total), not the exact sample values
 - the count of all observations (per bucket/total) 
 
 Therefore **we no longer have the ability to get the max value** from this metric as it is the histogram now (how do you get the max value if what you have is the already aggregated "sum of response times"?). The same ofc applies to all former value recorders. This is why we can't see the max plots in Grafana anymore in the example project setup.
 
 
What I propose is: 
 - merge this PR as is. I tested it and was able to see other metrics. I also see the histograms and the avg values seem to be calculated correctly from them
 - have an issue either to: 
   1. provide the max request/response value as a gauge (how feasible is that? Should we do this?)
   2. remove/change the plots in grafana to present something else
   
cc:  @jczuchnowski

Reply:
Ok, sounds reasonable. Let's merge and release, and talk about further improvements.